types/angular-ui-bootstrap/index.d.ts

```diff
@@ -510,15 +510,13 @@ declare module 'angular' {
             /**
              * Sets the `aria-describedby` property on the modal.
              * The string should be an id (without the leading '#') pointing to the element that describes your modal.
-             * @type {string}
```

Comment:
Removing `@type` is the most common change in the rule. This instance is a good exemplar - the type is inaccurate!